resources/stubs/modules/route-provider.stub

```diff
@@ -16,7 +16,7 @@ class RouteServiceProvider extends ServiceProvider
      *
      * @var string
      */
-    protected $namespace = '$MODULE_NAMESPACE$\$MODULE$\$CONTROLLER_NAMESPACE$';
+    protected $namespace = 'Modules\$MODULE$\Http\Controllers';
```

Comment:
Did `$MODULE_NAMESPACE$` not resolve? That's really strange

Reply:
It created namespace something like this :
```Modules\Module\Http\Controller```

It missed a 's' in Controllers spelling and also the arrangment of namespace wasn't clear if I recollect properly.

Reply:
I would leave `$MODULE_NAMESPACE$` if that was resolving properly. Let me pull this PR down in a bit and test that out. I don't like the laravel-modules plugin I'm using but I can't find another good one

Reply:
Haha! Yeah sure.
But I think there was some reason I removed Module namespace. 
Let me check it by putting $MODULE_NAMESPACE$ back.